tests/layouts/test_common_layout_features.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from textual.dom import NodeList
+from textual.screen import Screen
+from textual.widget import Widget
+
+
+@pytest.mark.parametrize(
+    "layout,display,expected_in_displayed_children",
+    [
+        ("dock", "block", True),
+        ("horizontal", "block", True),
+        ("vertical", "block", True),
+        ("dock", "none", False),
+        ("horizontal", "none", False),
+        ("vertical", "none", False),
```

Comment:
I noticed the existence of a `GridLayout`, but it's not managed in the layouts factory so I reckon it's a Work In Progress API?

Reply:
It's an older layout that I think we're scrapping